features/smoke-tests/buyer/catalogue.feature

```diff
@@ -15,14 +15,14 @@ Scenario: User can see the main links on the homepage
 
 @skip-local
 @skip-preview
+@skip-staging
```

Comment:
If this scenario is being skipped everywhere, can it be removed?

Reply:
It will be in the next revert

Reply:
It's still being run on production

Reply:
Oh wait, smoke tests....my bad,